src/Core/Grid/Definition/Factory/CurrencyGridDefinitionFactory.php

```diff
@@ -47,6 +47,8 @@
  */
 final class CurrencyGridDefinitionFactory extends AbstractGridDefinitionFactory
 {
+    use BulkDeleteActionTrait;
```

Comment:
@matks Why adding this trait if you not use it ?

Reply:
I might have done some overthinking 😅 my idea was "what if a module developer WANTS to extend the behavior of the Currency Grid by adding bulk delete"

Reply:
Removed